gravitee-apim-gateway/gravitee-apim-gateway-core/src/main/java/io/gravitee/gateway/core/processor/chain/AbstractStreamableProcessorChain.java

```diff
@@ -31,6 +34,8 @@ public abstract class AbstractStreamableProcessorChain<T, S, P extends Streamabl
     extends AbstractProcessorChain<T, P>
     implements StreamableProcessorChain<T, S, P> {
 
+    private final Logger logger = LoggerFactory.getLogger(EndpointGroupLifecycleManager.class);
```

Comment:
`AbstractStreamabledProcessorChain`?
I think you can also opt for lombok @Slf4j to avoid such copy-paste issue 🙂